Datagrid/SimplePager.php

```diff
@@ -43,6 +43,8 @@ class SimplePager extends Pager
      *
      * @param int $maxPerPage Number of records to display per page
      * @param int $threshold
+     *
+     * @deprecated Will be removed in version 2.0. Use \Sonata\AdminBundle\Datagrid\SimplePager instead.
```

Comment:
Please trigger a deprecation error and add `NEXT_MAJOR` instructions. Also, mark corresponding tests as legacy.